clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AbstractAvroStoreClient.java

```diff
@@ -75,6 +75,8 @@ public abstract class AbstractAvroStoreClient<K, V> extends InternalAvroStoreCli
   private RecordDeserializer<StreamingFooterRecordV1> streamingFooterRecordDeserializer;
   private TransportClient transportClient;
   private final Executor deserializationExecutor;
+  private final Executor veniceClientWarmUpExecutor =
+      Executors.newFixedThreadPool(1, new DaemonThreadFactory("Venice-Client-Warmup"));
```

Comment:
So we will allocate a single threaded pool for store client, can you append the store name to the thread pool name prefix?

It is fine to have a pool per store client as the warmup logic is only needed at startup time.

Reply:
Added store name.